controllers/suggestions/suggest_apiwrite_controller.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from consts.auth_type import AuthType
+from controllers.base_controller import LoggedInHandler
+from helpers.suggestions.suggestion_creator import SuggestionCreator
+from template_engine import jinja2_engine
+
+
+class SuggestApiWriteController(LoggedInHandler):
+    """
+    Allow users to request trusted API keys for an offseason event
+    """
+
+    def get(self):
+        self._require_login()
+        self.template_values.update({
+            "status": self.request.get("status"),
+            "auth_types": AuthType.type_names,
+        })
+        self.response.out.write(
+            jinja2_engine.render('suggest_apiwrite.html', self.template_values))
+
+    def post(self):
+        self._require_login()
+
+        status = SuggestionCreator.createApiWriteSuggestion(
+            author_account_key=self.user_bundle.account.key,
+            event_key=self.request.get("event_key", None),
+            affiliation=self.request.get("role", None),
+            auth_types=self.request.get_all("auth_types", [])
```

Comment:
I think this should sanitize `auth_types` to make sure they're in `AuthType.type_names.keys()` Or else I can request trusted API keys for an event in the future, spoof auth_types to include numbers we haven't yet defined, and if we do define them in the future then suddenly I have access.